evansmusomi/react-native-102
# Adds Launcher Icons

*What this does*
- Adds launcher icons

*How to verify*
- Uninstall the app and install it again
- You should see a new icon on the phone/emulator screen
